### ISSUE TITLE
NPM build breaking after latest commit

### ISSUE THREAD

**Issue by aem-octo** [Modernizr/customizr#70]
Hello
Thank your for your valuable open source work.
My project is build with node v6.14.1, npm 3.10.10, grunt 1.0.4 unfortunately i cannot update because of legacy.
The build started to fail 2 or three hours ago:
npm ERR! fetch failed https://github.com/Modernizr/customizr/tarball/develop
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404

Is this related to your latest commit ?

Thanks a lot and best regards,

**Comment by rejas** [comment]
The develop branch is no more, why are you using it at all?

**Comment by aem-octo** [comment]
Thanks for your quick reply, i really appreciate !
Honestly i don't know, i maintain a legacy project, i had this in package.json:
"grunt-modernizr": "~1.0.3",
I updated to:
"grunt-modernizr": "~2.0.1",
And now it builds OK

Does this makes sens to you ?

**Comment by rejas** [comment]
Not really.... Maybe that old release was tied to the develop branch which I deleted earlier this day. Happy that it works with the latest version. Anything else or can we close this issue?

**Comment by seeruk** [comment]
Also having this issue now... looks like we need to update.

**Comment by aem-octo** [comment]
>  Anything else or can we close this issue?
We don't have so much users on our website, so i won't have enough feedback quickly to confirm that service is OK. I can just confirm for now that build is 100% OK.
Maybe wait another feedback before closing the issue ?

Thanks again !

**Comment by coreyworrell** [comment]
> The develop branch is no more, why are you using it at all?

In our case, an old version of grunt-modernizr required it. Thankfully upgrading grunt-modernizr to newer version fixed it.
